src/components/PlaybackSpeedControl.tsx

```diff
@@ -0,0 +1,50 @@
+import { Slider } from "antd";
+import React, { ReactElement } from "react";
+
+import { DEFAULT_PLAYBACK_FPS } from "../constants";
+
+type PlaybackSpeedControlProps = {
+  fps: number;
+  onChange: (fps: number) => void;
+  baselineFps?: number;
+  disabled?: boolean;
+};
+const defaultProps: Partial<PlaybackSpeedControlProps> = {
+  fps: 30,
+  onChange: () => {},
+  baselineFps: DEFAULT_PLAYBACK_FPS,
+  disabled: false,
+};
+
+export default function PlaybackSpeedControl(inputProps: PlaybackSpeedControlProps): ReactElement {
+  const props = { ...defaultProps, ...inputProps } as Required<PlaybackSpeedControlProps>;
```

Comment:
you could, if you wanted, change the type of `defaultProps` from `Partial<_>` to `Required<_>` and avoid the cast on line 20

Reply:
I ended up just removing the two required props from the default ones!